clients/java/src/main/java/io/odpf/stencil/http/RetryHttpClient.java

```diff
@@ -26,6 +28,12 @@ public CloseableHttpClient create(StencilConfig stencilConfig) {
         long backoffMs = stencilConfig.getFetchBackoffMinMs() != 0 ? stencilConfig.getFetchBackoffMinMs() :
                 new RandomUtils().getRandomNumberInRange(DEFAULT_STENCIL_BACKOFF_MS_MIN, DEFAULT_STENCIL_BACKOFF_MS_MAX);
         int retries = stencilConfig.getFetchRetries();
+        List<Header> defaultHeaders = new ArrayList<>();
+
+        if (stencilConfig.getFetchAuthBearerToken() != null) {
```

Comment:
I was a bit b/w both of them and actually the thing is by default the value of this config is null. So if and only if the user has set this value, will this have a non null value. 
So basically if the developer using the library wants to set empty this will allow him to do it, instead of magically not setting it. wdyt?

Reply:
sure, even empty check doesn't change anything